src/java/install.sh

```diff
@@ -195,29 +195,29 @@ check_packages() {
     esac
 }
 
-# Use Microsoft JDK for everything but JDK 8 and 18 (unless specified differently with jdkDistro option)
-get_jdk_distro() {
-    VERSION="$1"
```

Comment:
I think we still need it, right? Look at test failures https://github.com/devcontainers/features/actions/runs/10472188406/job/29001078551?pr=1102#step:4:670